eng/mgmt/AzSdk.test.reference.props

```diff
@@ -3,8 +3,8 @@
 		<TargetFrameworks>netcoreapp2.0</TargetFrameworks>
 	</PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Test.HttpRecorder" Version="[1.13.3, 2.0.0)" />
-    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="[1.7.6, 2.0.0)" />
+    <PackageReference Include="Microsoft.Azure.Test.HttpRecorder" Version="[1.13.1, 2.0.0)" />
```

Comment:
We should update the versions in Packages.Data.props to match as opposed lowering this version.

Reply:
I did that, but it did not turn out well as it broke many of the DataPlane projects using those packages.